src/cpptools.ts

```diff
@@ -175,6 +175,10 @@ export interface CodeModelParams {
    * The active target
    */
   activeTarget: string|null;
+  /**
+   * Workspace folder.
+   */
+  folder: string
```

Comment:
Should we make this a Uri so that we know it's expected to be an absolute path?  If not, please add that to the comment.

Reply:
A Uri doesn't seem necessary and a caller using a relative path seems unlikely, but I can add more comment text.